Vixen.System/Module/Controller/ControllerProtocolTypes.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Net;
+
+namespace Vixen.Module.Controller
+{
+	public class ControllerNetworkConfiguration
+	{
+		public ProtocolTypes ProtocolType { get; set; }
+
+		public TransmissionMethods TransmissionMethod { get; set; }
+
+		public IPAddress IpAddress { get; set; }
+
+		//public string HostName { get; set; }
+		public bool SupportsUniverses { get; set; }
+		
+		public List<UniverseConfiguration> Universes { get; set; }
+	}
+
+	public class UniverseConfiguration

```

Comment:
This is not a problem, but I generally put each class in its own file as a good practice. It makes it easier to look at the file system and see what classes are there.